### PR TITLE
DEVTOOLS-98 Broke backward compatibility: Use the creation date when …

### PR DESCRIPTION
**Pull request by fschargorosdky**
…checking for conflicts

### CODE REVIEW

File: lib/services/localService.js [hunk 1]
```diff
@@ -119,19 +119,30 @@ module.exports = function (apiPlatformRepository, BPromise,
         localStatus.changed.forEach(function (changedFile) {
           var localFile = _.find(workspace.files, 'path', changedFile);
           var remoteFile = _.find(remoteFiles, 'path', changedFile);
-
-          var localFileLastUpdated =
-            localFile.audit && localFile.audit.updated ?
-            localFile.audit.updated.date :
-            undefined;
+          if (localFile.path === 'changedLocal') {
```
Comment (juancoen) [inline lib/services/localService.js]: this seems to be debugging code...

File: lib/services/localService.js [hunk 2]
```diff
@@ -119,19 +119,30 @@ module.exports = function (apiPlatformRepository, BPromise,
...
           }
+
+          function lastUpdated(file) {
+            if (file.audit) {
+              return file.audit.updated.date ?
+                file.audit.updated.date :
+                file.audit.created.date;
+            }
+
+            return undefined;
```
Comment (juancoen) [inline lib/services/localService.js]: no need to explicitely return undefined

File: lib/services/localService.js (hunk 1 above)
Comment (fschargorosdky) [inline lib/services/localService.js]: done

File: lib/services/localService.js (hunk 2 above)
Comment (fschargorosdky) [inline lib/services/localService.js]: done

**Comment by juancoen** [comment]
looks good to me

**Pull request merged**